### DISCUSSION TITLE
IA-32 + x86-64 Sun & oracle Solaris (and some illumos distros) failing to boot (+ UTM crash)

### DISCUSSION THREAD

**Discussion by MagnetarRocket** [utmapp/UTM#6328] [UTM Help (macOS)]
This had became annoying for me:

- "90's Solaris": 
  - Sun Solaris 2.4 on x86 fails to boot after updating to UTM 4.5.2 for when before it booted up to graphical install (even when after all of that, it couldn't boot from hard-disk). 
  - Sun Solaris 2.5.1 (the last Pentium-era version) does boot up to graphical install, and installs; but stuck in a loop. 
  - Sun Solaris 7 fails to boot at all.
  - Sun Solaris 8 automatically sees what i've had configured, but then goes into another loop.
- "Post-nevada (includes illumos distros)":
  - OpenSolaris 2009 does not boot beyond GRUB, don't know of a working config for such 21st century versions and above for x86 yet.
  - Tribblix does boot, but `installer.sh` is borked somehow.
  - OmniOS has very dark text, had to ramp up my M1's display just to see it. And it can't see the install disks at all.
  - SmartOS is very slow on first boot, install isn't user friendly so i couldn't see other problems.
  - Oracle Solaris refusing to boot at all, looked before and saw [*](https://github.com/utmapp/UTM/discussions/5777) , Apparently i'm not the only one left hanging on what is going wrong. I should also mention that after shutting down the emulated Oracle Solaris image- UTM crashes, often.
  - Only Post-Nevada system that works at all (so far) is OpenIndiana, and it's somewhat slow, especially with firefox.
  
Some of it is out of an need (I lost my OpenIndiana dev system apparently & now in a worse financial situation then when i still had the thing), and the 20th century solaris versions are more out of wanting to learn how openWindows flowed as NetBSD x86 didn't go far from `pkgin` & x86 slackware has repeated kernel panics (openLook so far aside from  is still a 32-bit only set of toolkits & libs, someone did a 64-bit version some years ago [*](https://github.com/ggodd/xview-64bit), but it seems like it's still linux exclusive & under a non-permissive license).

If there was a way to get satisfactory performance (and constant boots with non-OpenIndiana systems), i'd be happy to hear it.

**Comment by MagnetarRocket** [comment]
I wish i got Solaris 9 so that i could state the entire series, but alas no such thing as of late.

**Comment by joebb97** [comment]
Wanted to hop in here and say I'm seeing similar issues with OpenIndiana and OmniOS

Both get into the bootloader but then get stuck after that.

Tried both the pre-made VM disks and the CD-ROM ISOs but both fail the same way
